README.md

```diff
@@ -43,6 +43,27 @@ configured at run time to mimic specific real-world sensors. A noise model
+| **Air Pressure Sensor**   | [AirPressureSensor](https://gazebosim.org/api/sensors/9/classgz_1_1sensors_1_1AirPressureSensor.html)        | [SDF Air Pressure](http://sdformat.org/spec?ver=1.9&elem=sensor#sensor_air_pressure)              | [air_pressure.cc](https://github.com/gazebosim/gz-sensors/blob/main/test/integration/air_pressure.cc)                | Measures atmospheric pressure                      |
+| **Air Speed Sensor**      | [AirSpeedSensor](https://gazebosim.org/api/sensors/9/classgz_1_1sensors_1_1AirSpeedSensor.html)              | [SDF Air Speed](http://sdformat.org/spec?ver=1.9&elem=sensor#sensor_air_speed)                    | [air_speed.cc](https://github.com/gazebosim/gz-sensors/blob/main/test/integration/air_speed.cc)                      | Measures the speed of air relative to the sensor   |
+| **Altimeter**             | [AltimeterSensor](https://gazebosim.org/api/sensors/9/classgz_1_1sensors_1_1AltimeterSensor.html)            | [SDF Altimeter](http://sdformat.org/spec?ver=1.9&elem=sensor#sensor_altimeter)                    | [altimeter.cc](https://github.com/gazebosim/gz-sensors/blob/main/test/integration/altimeter.cc)                      | Measures altitude above a reference point          |
+| **Bounding Box Camera**   | [BoundingBoxCameraSensor](https://gazebosim.org/api/sensors/9/classgz_1_1sensors_1_1BoundingBoxCameraSensor.html) | [SDF Bounding Box Camera](http://sdformat.org/spec?ver=1.9&elem=sensor#sensor_boundingbox_camera) | [boundingbox_camera.cc](https://github.com/gazebosim/gz-sensors/blob/main/test/integration/boundingbox_camera.cc)    | Captures images with bounding box annotations      |
+| **Camera**                | [CameraSensor](https://gazebosim.org/api/sensors/9/classgz_1_1sensors_1_1CameraSensor.html)                  | [SDF Camera](http://sdformat.org/spec?ver=1.9&elem=sensor#sensor_camera)                          | [camera.cc](https://github.com/gazebosim/gz-sensors/blob/main/test/integration/camera.cc)                            | Captures standard RGB images                       |
+| **Depth Camera**          | [DepthCameraSensor](https://gazebosim.org/api/sensors/9/classgz_1_1sensors_1_1DepthCameraSensor.html)        | [SDF Depth Camera](http://sdformat.org/spec?ver=1.9&elem=sensor#sensor_depth_camera)              | [depth_camera.cc](https://github.com/gazebosim/gz-sensors/blob/main/test/integration/depth_camera.cc)                | Captures depth information                         |
+| **Doppler Velocity Log**  | [DopplerVelocityLog](https://gazebosim.org/api/sensors/9/classgz_1_1sensors_1_1DopplerVelocityLog.html)      | [SDF DVL](http://sdformat.org/spec?ver=1.9&elem=sensor#sensor_dvl)                                | [dvl.cc](https://github.com/gazebosim/gz-sensors/blob/main/test/integration/dvl.cc)                                  | Measures velocity relative to a water body         |
```

Comment:
the sensors produces velocity relative to seabed or by tracking water mass. I think we don't have to that detailed and just have a generic description note:

```suggestion
| **Doppler Velocity Log**  | [DopplerVelocityLog](https://gazebosim.org/api/sensors/9/classgz_1_1sensors_1_1DopplerVelocityLog.html)      | [SDF DVL](http://sdformat.org/spec?ver=1.9&elem=sensor#sensor_dvl)                                | [dvl.cc](https://github.com/gazebosim/gz-sensors/blob/main/test/integration/dvl.cc)                                  | Measures relative velocity of an underwater vehicle        |
```